ls1intum/Apollon
# Move zoom pane styles to styled component to ensure no issues when in…

This PR moves styles from the CSS file to a styled component as otherwise the version of Apollon integrated with Apollon Standalone will not apply styles to the zoom buttons as expected.

### Checklist

### Motivation and Context
The zoom buttons on the Apollon Canvas were not rendered properly in Apollon Standalone.

### Description
The styles used for the zoom buttons of the canvas are inlined using 'styled' components. The snapshot test regarding the canvas component are updated accordingly.

### Steps for Testing
As this change only changes how styles are applied, the easiest approach to test this PR would be by running Apollon and ensuring that the zoom buttons including their hover states look as expected.

### Screenshots
<img width="1157" alt="Screenshot 2023-10-12 at 14 05 08" src="https://github.com/ls1intum/Apollon/assets/143808484/9e3f8085-ba42-4ace-a383-61b931988285">
